_includes/payment-link.md

```diff
@@ -0,0 +1,238 @@
+    that is decided by the issuing bank.
+    Normally this will be done using BankID or Mobile BankID.
+{% endif %}
+
+```mermaid
+sequenceDiagram
+    activate Consumer
+    Consumer->>-MerchantOrderSystem: consumer starts purchase
+    activate MerchantOrderSystem
+    MerchantOrderSystem->>-Merchant: start purchase process
+    activate Merchant
+    Merchant->>-SwedbankPay: POST [payment] (operation=PURCHASE)
+    activate SwedbankPay
+    note left of Merchant: First API request
+    SwedbankPay-->>-Merchant: payment resource with payment URL
+    activate Merchant
+    Merchant-->>-MerchantOrderSystem: Payment URL sent to order system
+    activate MerchantOrderSystem
+    MerchantOrderSystem-->>-Consumer: Distribute Payment URL through e-mail/SMS
+    activate Consumer
+    note left of Consumer: Payment Link in e-mail/SMS
+    Consumer->>-SwedbankPay: Open link and enter payment information
+    activate SwedbankPay
+
+    {% if show_3d_secure %}
+        opt Card supports 3-D Secure
+        SwedbankPay-->>-Consumer: redirect to IssuingBank
+        activate Consumer
+        Consumer->>IssuingBank: 3-D Secure authentication process
+        Consumer->>-SwedbankPay: access authentication page
+        activate SwedbankPay
+        end
+    {% endif %}
+
+    SwedbankPay-->>-Consumer: redirect to merchant site
+    activate Consumer
+    note left of SwedbankPay: redirect back to merchant
+    Consumer->>-Merchant: access merchant page
+    activate Merchant
+    Merchant->>-SwedbankPay: GET [payment]
+    activate SwedbankPay
+    note left of Merchant: Second API request
+    SwedbankPay-->>-Merchant: payment resource
+    activate Merchant
+    Merchant-->>-Consumer: display purchase result
+```
+
+#### Options after posting a payment
+
+*   If the payment enable a two-phase flow (`Authorize`),
+    you will need to implement the `Capture` and `Cancel` requests.
+*   It is possible to "abort" the validity of the Payment Link.
+    [See the Abort description here][abort].
+*   For reversals, you will need to implement the `Reversal` request.
+*   When implementing the Payment Link scenario, it is optional to set a
+    `callbackURL` in the `POST` request.
+    If `callbackURL` is set Swedbank Pay will send a postback request to this
+    URL when the consumer as fulfilled the payment.
+    [See the Callback API description here][technical-reference-callback].
+
+[card-payment]: /assets/img/payments/card-payment.png
+[abort]: /payments/card/after-payment#abort
+[credit-card]: /payments/card
+[mobile-pay]: /payments/mobile-pay
+[payment-instruments-card-payment-pages]: /payments/card/
+[payment-instruments-mobilepay-payment-pages]: /payments/mobile-pay/
+[purchase-flow]: /payments/card/#purchase-flow
+[swish]: /payments/swish
+[technical-reference-callback]: #callback
+[test_purchase]: /assets/img/checkout/test-purchase.png
+[card-payment]: /assets/img/payments/card-payment.png
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-duplicate-definitions] Do not use definitions with the same identifier (227:1)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5659803)